src/sx/src/keyframes.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { isBrowser } from '@adeira/js';
+import stringify from 'json-stable-stringify';
+
+import hash from './hashStyle';
+import type { AllCSSPropertyTypes } from './css-properties/__generated__/AllCSSPropertyTypes';
+import styleCollector from './StyleCollector';
+import transformStyleName from './transformStyleName';
+import transformStyleValue from './transformValue';
+import { injectCSS } from './injectRuntimeStyles';
+
+type KeyFrames = {|
+  +from?: AllCSSPropertyTypes,
+  +to?: AllCSSPropertyTypes,
+  +[number]: AllCSSPropertyTypes, // percentages
+|};
+
+const extractStyles = (styles: AllCSSPropertyTypes) => {
+  let cssDefinition = '';
+  for (const [styleName, value] of Object.entries(styles)) {
+    const styleValue: string = (value: any);
```

Comment:
Can you please avoid the unsound `Object.entries`? This type casting doesn't seem to be correct. The values can be numbers as well for example.